tasks/main.yml

```diff
@@ -1,43 +1,65 @@
 ---
-# tasks file for clamav
+- name: Determine ClamAV package names and config file (Fedora)
+  set_fact:
+    package_names:
+      - clamav
+      - clamav-update
+  when: ansible_facts['distribution'] == 'Fedora'
 
-- name: Install ClamAV package
-  package: name=clamav-daemon state=present
+- name: Determine ClamAV package names and config file (Debian)
+  set_fact:
+    package_names:
+      - clamav-daemon
+  when: ansible_facts['distribution'] == 'Debian'
 
-- name: Modify ClamAV configuration
-  blockinfile:
-    path: /etc/clamav/clamd.conf
+- name: Install ClamAV packages
+  package:
+    name: '{{ package_names }}'
     state: present
-    block: |
-      ExcludePath ^/dev/
-      ExcludePath ^/proc/
-      ExcludePath ^/sys/
 
 - name: Install virus_scan cron job
-  copy: src="virus_scan.sh" dest="/etc/cron.weekly/virus_scan" mode=755
-
-# Unless you do this, systemd can sometimes get confused when you try
-# to start a service you just installed
-- name: Systemd daemon-reload
-  systemd:
-    daemon_reload: true
-  when:
-    - ansible_service_mgr == "systemd"
-
-- name: Set freshclam to start at boot
-  service:
-    name: clamav-freshclam
-    enabled: yes
-    state: started
+  copy:
+    src: virus_scan.sh
+    dest: /etc/cron.weekly/virus_scan
```

Comment:
@felddy, is there a reason why we change the script's name when copying it over?  If not then I'd prefer to keep it the same.

Reply:
I agree.  Keep it the same.